tests/pytests/unit/modules/test_postgres.py

```diff
@@ -181,3 +212,2339 @@ def test__run_initdb_with_timeout():
             with patch.dict(configmod.__opts__, {"postgres.pass": None}):
                 postgres._run_initdb("fakename", runas="saltuser")
                 cmd_run_mock.assert_called_with(cmd_str, timeout=0, **kwargs)
+
+
+def test_run_psql():
+    postgres._run_psql('echo "hi"')
+    cmd = postgres.__salt__["cmd.run_all"]
+    assert cmd.call_args[1]["runas"] == "postgres"
+
+
+def test_db_alter():
+    with patch(
+        "salt.modules.postgres._run_psql", Mock(return_value={"retcode": 0})
+    ), patch("salt.utils.path.which", MagicMock(return_value="/usr/bin/pgsql")):
+        ret = postgres.db_alter(
+            "dbname",
+            user="testuser",
+            host="testhost",
+            port="testport",
+            maintenance_db="maint_db",
+            password="foo",
+            tablespace="testspace",
+            owner="otheruser",
+            runas="foo",
+        )
+        assert ret
```

Comment:
```suggestion
        assert ret is True
```

If we are expecting the return to be a boolean of True we should be explicit about this, or else returning other types of data would pass this test.

note: This applies to other parts of your PR as well where you're setting `assert ret`

Reply:
Actually the pytests doc's state 'assert ret' for something being True, so I was following the pytest docs, but of course when I go to find the reference to show you, I cannot find it :facepalm: .  But that is why I have the 'assert boolean' for testing if True.  Will dig around some more in pytest docs

Reply:
Not able to find it, but only did it that way since read it in the doc's, have no problem changing this to **assert boolean is True** format, was just trying to follow best-practice for pytest.
Will update tests